### DISCUSSION TITLE
宝塔部署node项目，一启动就自动停止

### DISCUSSION THREAD

**Discussion by Nextjsoul** [ddiu8081/chatgpt-demo#85] [Q&A]
![gpt宝塔](https://user-images.githubusercontent.com/125454476/222935365-33c96b76-419e-498c-9ff3-44a8d09c3d48.jpg)
配置如图，一启动就自动停止

**Comment by tankren** [comment]
node好像要18+

**Reply by Nextjsoul** [reply]
node.js版本换成18. 启动失败 启动失败

novel-gpt@0.0.1 dev > astro dev > novel-gpt@0.0.1 dev > astro dev > novel-gpt@0.0.1 dev > astro dev > novel-gpt@0.0.1 dev > astro dev > novel-gpt@0.0.1 dev > astro dev
